I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/dialogues/firstDay.rpy", line 7, in script
    monika "Siema nowy"
  File "renpy/common/00action_other.rpy", line 586, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/script.rpy", line 23, in goLeft
    renpy.call_screen("buttons")
Exception: Cannot start an interaction in the middle of an interaction, without creating a new context.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/dialogues/firstDay.rpy", line 7, in script
    monika "Siema nowy"
  File "D:\renpy\renpy-8.2.0-sdk\renpy\ast.py", line 2557, in execute
    Say.execute(self)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\ast.py", line 615, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\exports.py", line 1493, in say
    who(what, *args, **kwargs)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\character.py", line 1391, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\character.py", line 1042, in do_display
    display_say(who,
  File "D:\renpy\renpy-8.2.0-sdk\renpy\character.py", line 738, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\display\core.py", line 2165, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "D:\renpy\renpy-8.2.0-sdk\renpy\display\core.py", line 3197, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\display\layout.py", line 1202, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\display\layout.py", line 1202, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\display\layout.py", line 1202, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\display\screen.py", line 793, in event
    rv = self.child.event(ev, x, y, st)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\display\layout.py", line 1202, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\display\behavior.py", line 1174, in event
    return handle_click(self.clicked)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\display\behavior.py", line 1095, in handle_click
    rv = run(action)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\display\behavior.py", line 395, in run
    return action(*args, **kwargs)
  File "renpy/common/00action_other.rpy", line 586, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/script.rpy", line 23, in goLeft
    renpy.call_screen("buttons")
  File "D:\renpy\renpy-8.2.0-sdk\renpy\exports.py", line 3413, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\display\core.py", line 2134, in interact
    raise Exception("Cannot start an interaction in the middle of an interaction, without creating a new context.")
Exception: Cannot start an interaction in the middle of an interaction, without creating a new context.

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.0.24012702
Staszic The Game 1.0
Tue Mar 12 17:30:31 2024
